AssetInformationApi/V1/Controllers/AssetInformationApiController.cs

```diff
@@ -78,19 +79,16 @@ public async Task<IActionResult> GetAssetByAssetId([FromRoute] GetAssetByAssetId
             return Ok(result);
         }
 
-        [ProducesResponseType(typeof(AssetResponseObject), StatusCodes.Status200OK)]
-        [ProducesResponseType(StatusCodes.Status400BadRequest)]
-        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(AssetResponseObject), StatusCodes.Status201Created)]
```

Comment:
Should [ProducesResponseType(StatusCodes.Status400BadRequest)] still be present as 400 bad request is a valid status code if validation fails?

Reply:
I was just following what is in Tenure and Person apis. I have re-added it now.